recipes/qutip/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+    name: qutip
+    version: 3.1.0
+
+source:
+    fn: qutip-3.1.0.tar.gz
+    url: http://qutip.org/downloads/3.1.0/qutip-3.1.0.tar.gz
+
+build:
+    number: 1
+
+requirements:
+  build:
+    - setuptools
+    - python
+    - cython >=0.15
+    - numpy >=1.7
+    
+  run:
+    - python >=2.7
+    - cython >=0.15
+    - numpy >=1.7
```

Comment:
Does the package have compiled Cython components which depend on numpy? If so, it is likely they depend on the numpy ABI, which should be encoded into the recipe in the form:

```
 - numpy x.x
```

That will bake in the dependence of the numpy which it was built against (many builds will take place, one for each minor numpy version).